problems/removeVowels.js

```diff
@@ -6,8 +6,24 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+const removeVowels = (str) => {
+    let newStr = ""
+    for(let i =0; i < str.length;i++){
+        if(str.length === aeiouAeiou){
```

Comment:
You've got a lot of good ideas on this problem, but the execution is off. Currently aeiouAeiou is going to be an undeclared variable and throw an error. You want to make a variable with vowels as a string:
const vowels = "AEIOUaeiou"
Then instead of seeing if the length of a string is equal to it, we want to check if the character is included in our vowels.